owasp-top10-2017-apps/a8/amarelo-designs/deployments/docker-compose.yml

```diff
@@ -8,7 +8,7 @@ services:
     volumes:
       - "../app/:/app"
     ports:
-      - "5000:5000"
+      - "10008:10008"
```

Comment:
Flask in this app runs on port `5000` inside the container, so we need to forward it to `10008` outside to access it.

Do you mind changing this line to `10008:5000`?